assets/src/blocks/SocialMedia/SocialMediaScript.js

```diff
@@ -0,0 +1,10 @@
+import { SocialMediaFrontend } from './SocialMediaFrontend';
```

Comment:
If I understand correctly, we would only add this script for older blocks, as the newer ones are already rendered and saved. Perhaps it's easier (and better for performance) if we, instead of adding a new front end script, re purpose the existing back end code and Twig template?